pkgs/by-name/bu/bubblejail/env-python.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/templates/bubblejail_launch_base.py.jinja b/templates/bubblejail_launch_base.py.jinja
+index af91508..2dc9084 100644
+--- a/templates/bubblejail_launch_base.py.jinja
++++ b/templates/bubblejail_launch_base.py.jinja
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python3 -{{ PYTHON_SHEBANG_SHORT_OPTS }}
++#!/usr/bin/env python3 -{{ PYTHON_SHEBANG_SHORT_OPTS }}
```

Comment:
I don't think this will work without `-S` flag. (`#!/usr/bin/env -S python3 -{{ PYTHON_SHEBANG_SHORT_OPTS }}`) Otherwise it will be treated as one long argument.